package.json

```diff
@@ -87,7 +87,7 @@
   ],
   "scripts": {
     "pretest": "test -f logs/test.log && rm logs/test.log || echo 'No test log to remove'",
-    "test": "gulp test"
+    "test": "mocha --full-trace --timeout 10000 node_modules/col-tests/lib/beforeTests.js node_modules/col-*/tests/**/*.js"
```

Comment:
@paulkerschen I did not prepend `NODE_ENV=test` in line above because it's used by travis, too.